lib/looker.ts

```diff
@@ -0,0 +1,53 @@
+import { IDashboardElement, IWriteQuery } from "@looker/sdk"
+import { SDK } from "./sdk";
+
+export async function getAWDashboardElement0(): Promise<IDashboardElement> {
+  const dashboardId = "1471";
```

Comment:
Don't necessarily need to fix this now, since we've spun off a bug, but this is why the Infobar CTR stuff isn't working, in part.  We have a different dashboard ID for infobars (in addition to have a different JSON property name or two in the results).